motor/providers/container/container.go

```diff
@@ -110,14 +113,28 @@ func NewDockerEngineImage(endpoint *providers.Config) (ContainerProvider, error)
 		return nil, err
 	}
 
+	labelImageIdFull := strings.Split(ii.Labels["docker.io/digests"], ",")[0]
+	labelImageId := strings.Split(labelImageIdFull, "@")[1]
+
 	log.Debug().Msg("found docker engine image " + ii.ID)
+	// This is the image id that is used to pull the image from the registry.
+	log.Debug().Msg("found docker engine image " + labelImageId)
+	if ii.Size > 1024 { // > 1GB
+		log.Warn().Int64("size", ii.Size).Msg("image is bigger than 1GB, this will take some time")
+	}
 	img, rc, err := image.LoadImageFromDockerEngine(ii.ID)
 	if err != nil {
 		return nil, err
 	}
 
 	var identifier string
+	// This is not the digest used to pull the image from the registry.
```

Comment:
IIRC I used that to filter out duplicate container images in k8s. If that's not the digest used to pull the image, then the function is used in the wrong way. If we can use the actual digest it would be good. I can't see a reason to use this `img.Digest()`